### PR TITLE
util-misc: Convert unittests to new FAIL/PASS API - v3

### PR DESCRIPTION
**Pull request by hadiqaalamdar**
Task #6345

Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [x] I have read the contributing guide lines at https://docs.suricata.io/en/latest/devguide/codebase/contributing/contribution-process.html
- [x] I have signed the Open Information Security Foundation contribution agreement at https://suricata.io/about/contribution-agreement/
- [x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/issues/6345) ticket:
Previous PR https://github.com/OISF/suricata/pull/9656

Describe changes:
- Update `UtilMiscParseSizeStringTest01` in util-misc unit tests

### CODE REVIEW

**Review by jufajardini — CHANGES_REQUESTED** [review]
Hi there, quick feedback: we use the version for branch names and PRs, but not the for commits themselves! :P

File: .DS_Store
Comment (jufajardini) [inline .DS_Store:1]: I believe this was added by mistake?

Comment (hadiqaalamdar) [reply .DS_Store:1]: Yes, my apologies. Should I change the commit name?

Comment (jufajardini) [reply .DS_Store:1]: Yes, please!! And, of course, remove the unwanted file ;)

Comment (hadiqaalamdar) [reply .DS_Store:1]: should I create a new PR for this or make change to this one?

Comment (jufajardini) [reply .DS_Store:1]: New PR is the preferred way, please :)

**Comment by hadiqaalamdar** [comment]
Followed by https://github.com/OISF/suricata/pull/9663

**Pull request closed without merging**